### ISSUE TITLE
No Implementation for `Be<u16> |= u16`

### ISSUE THREAD

**Issue by hftsai256** [wrenger/bitfield-struct-rs#52]
bitfield-struct version: 0.9.0
rustc version: 1.81.0

#### Symptom
The error will show up when a padding field (with a leading underscore) presents, or a read-only field is configured in the struct

#### Minimum Reproducing Example:
```
#[bitfield(u16, repr = le16, from = le16::from_ne, into = le16::to_ne)]
struct Test {
    #[bits(15)]
    f3: u16,
    // _reserved: bool  // will not compile
    // #[bits(1, access = RO)]  // will not compile
    reserved: bool  // can compile
}
```

**Comment by wrenger** [comment]
Oh interesting, I have to look into that.

**Comment by wrenger** [comment]
Ok I've found and fixed the issue in 3f48bdc573f3b230330253d3215f15fb8a567ffc, which will be part of 0.9.1
